diff --git a/mmdet3d/core/evaluation/__init__.py b/mmdet3d/core/evaluation/__init__.py
index b1d489f..939e90b 100644
--- a/mmdet3d/core/evaluation/__init__.py
+++ b/mmdet3d/core/evaluation/__init__.py
@@ -5 +4,0 @@ from .kitti_utils import kitti_eval, kitti_eval_coco_style
-from .lyft_eval import lyft_eval
@@ -9 +8 @@ __all__ = [
-    'kitti_eval_coco_style', 'kitti_eval', 'indoor_eval', 'lyft_eval',
+    'kitti_eval_coco_style', 'kitti_eval', 'indoor_eval',
diff --git a/mmdet3d/datasets/__init__.py b/mmdet3d/datasets/__init__.py
index dab7502..ae58b25 100644
--- a/mmdet3d/datasets/__init__.py
+++ b/mmdet3d/datasets/__init__.py
@@ -8 +7,0 @@ from .kitti_mono_dataset import KittiMonoDataset
-from .lyft_dataset import LyftDataset
@@ -34 +33 @@ __all__ = [
-    'build_dataset', 'NuScenesDataset', 'NuScenesMonoDataset', 'LyftDataset',
+    'build_dataset', 'NuScenesDataset', 'NuScenesMonoDataset',
diff --git a/requirements/runtime.txt b/requirements/runtime.txt
index 643cb0c..73dac63 100644
--- a/requirements/runtime.txt
+++ b/requirements/runtime.txt
@@ -1,3 +1,2 @@
-lyft_dataset_sdk
-networkx>=2.2,<2.3
-numba==0.53.0
+networkx==2.8.8
+numba==0.60.0
@@ -6,4 +5,0 @@ nuscenes-devkit
-plyfile
-scikit-image
-# by default we also use tensorboard to log results
-tensorboard
